store/cart.ts

```diff
@@ -1,118 +1,34 @@
-import { Module } from 'vuex'
-import { CartState } from '../types/Cart'
+export const actions = {
+  async getCart({ rootState, commit }) {
```

Comment:
```suggestion
  async fetchCart({ rootState, commit }) {
```

Would recommend renaming this to `fetchCart` since `getCart` would imply that it's a getter

Reply:
Agree.

I saw that you used fetch as the function name, but that confused me because there is both old js fetch and new fetch hook from nuxt.

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch
https://nuxtjs.org/api/pages-fetch/

Reply:
changed to fetchCart in latest commit